settings.py

```diff
@@ -11,6 +11,7 @@
     parsed_vcap_services = json.loads(vcap_services)
     rds_config = parsed_vcap_services.get('rds')
     DB_URI = rds_config[0].get('credentials').get('uri')
+    RESPONSE_RETENTION_DAYS = 90  # Defaulted in CF, (never probably going to be used), but a reminder for if we ever do
```

Comment:
You don't need to define this in 2 separate places.  This can be a singlular `RESPONSE_RETENTION_DAYS = os.getenv('SDX_STORE_RESPONSE_RETENTION_DAYS')` outside of this if/else block and it'd work just as well for both of them

Reply:
ok ,changed